### DISCUSSION TITLE
Is there a way of accessing `GIT_REV` during the build phase in the Dockerfile?

### DISCUSSION THREAD

**Discussion by eillarra** [dokku/dokku#5901] [Q&A] [answered]
I have tried with

```Dockerfile
ARG GIT_REV
ENV GIT_REV ${GIT_REV}
```

but the environment variable is not set yet, and I couldn't figure out a way of including it via `--build-arg`, as it should be changing with every deploy. Cheers!

**Comment by josegonzalez** [comment]
Does setting `--build-arg GIT_REV` without a value as a build-time docker option work?

**Comment by eillarra** [comment]
No, it doesn't. This is what I have done, but the variable is not available:

```shell
dokku docker-options:add appname build "--build-arg GIT_REV"
```

**Reply by josegonzalez** [reply, marked as answer]
We can probably make an enhancement here to make the env var available so the build-arg works. I'd recommend filing a ticket :)

**Comment by theronic** [comment]
Any update on this? Also need this please :)

**Reply by josegonzalez** [reply]
It will land in the next minor: https://github.com/dokku/dokku/pull/5947
